nss-day-cohort-46/reactive-nutshell-talisman-of-havocing
# Nc the actual event component

# Description
The event list renders a list of saved events by order of date. Earliest on top. You can add an event in the form and it will show up on the list.

## Type of change


# Testing Instructions
move the data for the events from the .example file into your json file. Then see that they render to the event list. Try creating an event. See that they render in the correct order.

# Checklist:
